interface/main/calendar/modules/PostCalendar/pnadmin.php

```diff
@@ -700,52 +700,52 @@ function postcalendar_adminmenu($menuItem)
 
     if ($menuItem === "clearCache") {
         $output .= <<<EOF
-<li class="nav-item">
-    <a class="nav-link active" href="$cacheURL">$cacheText</a>
-</li>
-<li class="nav-item">
-    <a class="nav-link" href="$systemURL">$systemText</a>
-</li>
-<li class="nav-item">
-    <a class="nav-link" href="$categoryURL">$categoryText</a>
-</li>
-EOF;
+        <li class="nav-item">
+            <a class="nav-link active" href="$cacheURL">$cacheText</a>
+        </li>
+        <li class="nav-item">
+            <a class="nav-link" href="$systemURL">$systemText</a>
+        </li>
+        <li class="nav-item">
+            <a class="nav-link" href="$categoryURL">$categoryText</a>
+        </li>
+        EOF;
```

Comment:
Indentation is a good idea but IMHO placing `EOF;` without indentation will make it more noticable.
@bradymiller what are your thoughts about this?

Reply:
This is actually part of a renegade commit and will be going away after branch gets cleaned up per my instructions above. Trying to indent EOF blocks here actually causes parsing error in PHP 7.2 (this is why travis is failing).